packages/frontend-api/apiary.apib

```diff
@@ -9,48 +9,54 @@ HOST: https://polls.apiblueprint.org/
 
 Returns list of product that can be paginated using `first`, `last`, `before` and `after` keywords.
 You can read more about Connection specification in [connections article](https://relay.dev/graphql/connections.htm).
+If you use the authorization header, api will return prices according to the user's price group.
+Authorization header is not required.
 
 - Request (application/json)
 
-    {
-        query: products (first: 2) {
-            edges {
-                node {
-                    uuid
-                    name
-                    shortDescription
-                    link
-                    unit {
-                        name
-                    }
-                    availability {
-                        name
-                    }
-                    stockQuantity
-                    categories {
-                        name
-                    }
-                    flags {
+    - Headers
+        Authorization: Bearer ABCDEF
```

Comment:
I would use access token from login method example

Reply:
I don't think so because the token is not a nice format and it wouldn't look nice in the documentation :smile:

Reply:
b46de52fdd97b7c73b5d49e25245679a7b3e8886